cpu/sam0_common/periph/i2c.c

```diff
@@ -11,7 +11,7 @@
  * @{
  * @file
  * @brief       Low-level I2C driver implementation
- * @author      Baptiste Clenet <baptiste.clenet@xsoen.com>
+ * @author      Baptiste Clenet <bapclenet@gmail.com>
```

Comment:
The problem is that it touches very different files (in sam0_common, samd21 and saml21). Can you open a dedicated PR for these changes ?